.github/workflows/check-entangled-specs.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       - name: Setup Python 3.7
```

Comment:
Can you update the names to match the version, or just change them to generic "Setup Python"?

Reply:
Ah, right, good catch.:)